scripts/buildTs.mjs

```diff
@@ -7,18 +7,18 @@
 
 'use strict';
 
-const assert = require('assert');
-const fs = require('fs');
-const os = require('os');
-const path = require('path');
-const chalk = require('chalk');
-const execa = require('execa');
-const globby = require('globby');
-const stripJsonComments = require('strip-json-comments');
-const throat = require('throat');
-const {getPackages} = require('./buildUtils');
-
-(async () => {
```

Comment:
TLA is only supported on node 14 and later, I think the wrapper is still needed (even if CI doesn't call this script on node 12)